.github/workflows/e2e_libvirt.yaml

```diff
@@ -105,6 +105,12 @@ jobs:
           git checkout "${KBS_VERSION}"
           pushd kbs
           make CLI_FEATURES=sample_only cli
+          pushd config/kubernetes/base/
+          # Trustee only updates their staging image reliably with sha tags,
+          # so switch to use that and convert the version to the sha
+          KBS_SHA=$(gh api repos/confidential-containers/trustee/commits/${KBS_VERSION} -q .sha)
+          kustomize edit set image kbs-container-image=ghcr.io/confidential-containers/staged-images/kbs:${KBS_SHA}
```

Comment:
Hi @stevenhorsman !

Allow me to ask you 2 changes here:

1) cat the kustomization file, for debugging sake
2) Grep for `ghcr.io/confidential-containers/staged-images/kbs` and `${KBS_SHA}`. In some places in kata we have this safeguard, in others not. The problem is that `kustomize` may silently not set the image (e.g. in case `kbs-container-image` change), and usually it takes ages until it's realized we were using a wrong image.

Reply:
For part 2 do you mean in the workflow output:
```
$ cat kustomization.yaml | grep -A 3 images:
images:
- name: kbs-container-image
  newName: ghcr.io/confidential-containers/key-broker-service
  newTag: test
```
for debug, or something else?